ivadomed/scripts/visualize_transforms.py

```diff
@@ -81,36 +81,37 @@ def run_visualization(fname_input, fname_config, n_slices, folder_output, fname_
         :align: center
 
     Args:
-         fname_input (string): Image filename.
-         fname_config (string): Configuration file filename.
-         n_slices (int): Number of slices randomly extracted.
-         folder_output (string): Folder path where the results are saved.
-         fname_roi (string): Filename of the region of interest. Only needed if ROICrop is part of the transformations.
+         input (string): Image filename. Flag: --input, -i
+         config (string): Configuration file filename. Flag: --config, -c
+         number (int): Number of slices randomly extracted. Flag: --number, -n
+         ofolder (string): Folder path where the results are saved. Flag: --ofolder, -o
```

Comment:
ofolder --> output in order to match the other functions? not sure if important, your call.

Reply:
for the sake of consistency, it could be a good idea I'll change it !

Reply:
Done in bd572a6